src/server/StudioManagementSystem/ProjectManagement/GroupManagement.cs

```diff
@@ -0,0 +1,38 @@
+﻿using StudioManagementSystem.Core.Dtos;
+using StudioManagementSystem.Infrastructure.Interfaces.Data;
+using StudioManagementSystem.Mappers;
+
+namespace StudioManagementSystem.ProjectManagement;
+
+public interface IProjectGroupManager
+{
+    Task<Guid> CreateNewGroupAsync(CreateGroupDto dto, CancellationToken ct);
+}
+
+[InstanceScopedBusinessService]
+public class ProjectGroupManager : IProjectGroupManager
+{
+    private readonly IGroupRepository _groupRepository;
+
+    public ProjectGroupManager(IGroupRepository groupRepository)
+    {
+        _groupRepository = groupRepository;
+    }
+
+    public async Task<Guid> CreateNewGroupAsync(CreateGroupDto dto, CancellationToken ct)
+    {
+        var groupId = await _groupRepository.AddGroupAsync(new(dto), ct);
+        var group = await _groupRepository.GetGroupAsync(groupId, ct);
+
+        var groupProjectPreferences = dto.Preferences
+            .PreferencesAsList()
+            .Select(GroupProjectPreferenceMapper.MapToGroupProjectPreference)
+            .ToList();
+
+        if (group != null) {
+            await _groupRepository.AddGroupProjectPreferencesAsync(groupProjectPreferences, group, ct);
+        }
+
+        return groupId;
+    }
```

Comment:
^ this work will be refactored/extended to support individual project preferences

@Ryan-Cleminson this will require spec'ing out a task to define how we want to assign projects to students. At this point we have data to,

* create projects and groups
* assign project owners
* assign (ranked) project preferences to groups

Next up is assigning students to a project, which unblocks adding the preference